### PR TITLE
SISRP-7763 Make sis-qa-test-auto run in vagrant

### PR DESCRIPTION
**Pull request by mDibyo**
https://jira.berkeley.edu/browse/SISRP-7763

### CODE REVIEW

**Comment by mDibyo** [comment]
Please check that the Vagrantfile works on a Mac as well.

**Comment by christianvuerings** [comment]
@mDibyo Since CalCentral already runs on 3000 by default, this was giving me some errors in the beginning. Could we use a different port instead? E.g. `3300`?

File: README.md [hunk 1]
```diff
@@ -62,6 +62,13 @@ Engine](https://docs.docker.com/installation/)).
 and/or logs directory, run as `sudo -E docker-compose up` to pass in environment variables). The
 first run will take longer since the Docker images will be built and then run.
 
+### Using Vagrant
+This method runs Docker Engine inside a Vagrant VM.
+
+1. In the project directory, run `vagrant up`. This will also build the Docker images inside the
+VM and set all environment variables.
+1. `vagrant ssh` into the VM and then start the service with `sudo -E qata start`.
```
Comment (christianvuerings) [inline README.md:70]: @mDibyo I can confirm that everything worked correctly and was able to access the dashboard on Mac. Great job!
Would love to start showing this on a monitor in our office

**Comment by mDibyo** [comment]
@christianvuerings Sure! I will change the default port for the dashboard to 3300.

File: README.md (hunk 1 above)
Comment (mDibyo) [inline README.md:70]: Thanks!

**Comment by mDibyo** [comment]
I am hoping this will make the review process for future PRs easier.

Also, @christianvuerings, would it be possible to use this Vagrantfile as the image for the test VM?

**Comment by christianvuerings** [comment]
@mDibyo that would be fine with me. Do you need help with that or is it something you could do by yourself?

**Pull request merged**